### PR TITLE
Unit test added for Temporal utilities class

### PR DESCRIPTION
**Pull request by Quix0r**
Changes:
- added unit-test for `Temporal::getRelativeDate()` method

### CODE REVIEW

File: tests/src/Util/TemporalTest.php [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+		// Format current date/time into "MySQL" format
+		$now = date('Y-m-d H:i:s');
+		self::assertEquals(
+			Temporal::getRelativeDate($now),
+			DI::l10n()->t('less than a second ago')
+		);
+
+		// Format current date/time - 1 minute into "MySQL" format
+		$minuteAgo = date('Y-m-d H:i:s', time() - 60);
+		$format = DI::l10n()->t('%1$d %2$s ago');
```
Comment (MrPetovan) [inline tests/src/Util/TemporalTest.php]: Huh, this tells me we're doing the localization labels wrong, they should use `L10n->tt()` instead, with a different string for each time unit.

**Review by MrPetovan — APPROVED** [review]

**Comment by MrPetovan** [comment]
Please address the code standards check.

**Comment by Quix0r** [comment]
I have added `./bin/dev/php-cs-fixer/vendor/bin/php-cs-fixer fix $CHANGED_FILE --dry-run` to my pre-commit hook to prevent this from happening.

File: tests/src/Util/TemporalTest.php (hunk 1 above)
Comment (Quix0r) [reply tests/src/Util/TemporalTest.php]: Is there anything on my side I can change?

**Review by MrPetovan — APPROVED** [review]

File: tests/src/Util/TemporalTest.php (hunk 1 above)
Comment (MrPetovan) [reply tests/src/Util/TemporalTest.php]: No, it's an invasive change outside of the scope of this PR. You are welcome to do it in a separate one though!

**Pull request merged**